.github/workflows/main.yml

```diff
@@ -0,0 +1,69 @@
+name: h5z-zfp
```

Comment:
Would you recommend I replace travis with this GH actions?

Reply:
For CGNS, we had to move from travisCI because we had used all our free hours. I think Github actions is better and I like how it is integrated into Github.